src/PVRFilmonData.cpp

```diff
@@ -333,6 +359,27 @@ PVR_ERROR PVRFilmonData::GetRecordings(ADDON_HANDLE handle) {
 	return PVR_ERROR_NO_ERROR;
 }
 
+PVR_ERROR PVRFilmonData::GetRecordingStreamProperties(const PVR_RECORDING* recording, PVR_NAMED_VALUE* properties, unsigned int* iPropertiesCount) {
+  P8PLATFORM::CLockObject lock(m_mutex);
+  std:string strRecordingFile = "";
```

Comment:
BTW, no need to initialize the string with an empty string, because this what the default constructor does automatically. Applies as well to other parts of this PR.